README.md

```diff
@@ -41,7 +44,7 @@ on:
 
 jobs:
   prepare-release:
-    uses: heroku/languages-github-actions/.github/workflows/_buildpacks-prepare-release.yml@main
+    uses: heroku/languages-github-actions/.github/workflows/_buildpacks-prepare-release.yml@latest
     with:
       app_id: ${{ vars.GH_APP_ID }}
```

Comment:
I presume this is `vars.GH_APP_ID` instead of `vars.LINGUIST_GH_APP_ID` (and similar for the secrets section) since you're wanting to keep the README generic, and not tied to our specific Linguist implementation choices? (I'm happy with either approach, so how it is at the moment is fine - I'm only mentioning in case this naming wasn't intentional)

Reply:
yeah, it was to keep the README generic